gabbi/runner.py

```diff
@@ -213,6 +222,11 @@ def _make_argparser():
         action='store_true',
         help='Exit on first failure'
     )
+    parser.add_argument(
+        '-q', '--quiet',
+        action='store_true',
+        help='Produce no test runner output'
```

Comment:
That phrasing seems a little awkward; "Suppress test runner output" seems more straightforward?